packages/documentation/src/stories/components/alert/standard-html/alert.stories.ts

```diff
@@ -17,6 +18,10 @@ const meta: MetaComponent = {
     controls: {
       exclude: ['Title', 'Content'],
     },
+    design: {
+      type: 'figma',
+      url: getComponentFigmaLink('17001-2244'),
+    },
```

Comment:
For compatibility, I used the same schema as official figma addons (even if it's rather per story than per docs page): the https://storybook.js.org/docs/sharing/design-integrations#link-figma-components-to-stories